test/test-network/systemd-networkd-tests.py

```diff
@@ -933,6 +933,17 @@ def test_altname(self):
         output = check_output(*networkctl_cmd, '-n', '0', 'status', 'dummy98', env=env)
         self.assertRegex(output, 'hogehogehogehogehogehoge')
 
+    @expectedFailureIfAlternativeNameIsNotAvailable()
+    def test_rename_to_altname(self):
+        copy_network_unit('26-netdev-link-local-addressing-yes.network',
+                          '12-dummy.netdev', '12-dummy-rename-to-altname.link')
+        start_networkd()
+        self.wait_online(['dummyalt:degraded'])
+
+        output = check_output(*networkctl_cmd, '-n', '0', 'status', 'dummyalt', env=env)
+        self.assertRegex(output, 'hogehogehogehogehogehoge')
+        self.assertNotRegex(output, 'dummy98')
```

Comment:
Not a big matter, but please use `assertIn()` and `assertNotIn()` if no regex is used.